client/app/scripts/superdesk-archive/module.js

```diff
@@ -94,7 +94,10 @@ define([
                         });
                     }],
                     filters: [{action: 'list', type: 'archive'}],
-                    action: 'spike'
+                    action: 'spike',
+                    condition: function(item) {
+                        return item.lock_user === null || angular.isUndefined(item.lock_user);
```

Comment:
fyi you can also use `item.lock_user != null`, it works for both `undefined` and `null`